sympy/solvers/inequalities.py

```diff
@@ -126,7 +126,7 @@ def solve_poly_inequalities(polys):
     Union(Interval.open(-oo, -sqrt(3)), Interval.open(-1, 1), Interval.open(sqrt(3), oo))
     """
     from sympy import Union
-    return Union(*[solve_poly_inequality(*p) for p in polys])
+    return Union(*sum([solve_poly_inequality(*p) for p in polys],[]))
```

Comment:
Using `sum` to combine lists is bad, because it has quadratic performance. You should be able to get the same thing with `Union(*[i for p in polys for i in solve_poly_inequality(*p)])`.

Reply:
I didn't know that. I'll change it to your suggestion. But I picked up the syntax only from `Union()` constructor in `sets.py`. There `sum` is used along with `map`